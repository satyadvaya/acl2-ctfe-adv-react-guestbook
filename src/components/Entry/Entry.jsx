import React from 'react';

export default function Entry({ entry: { name, message } }) {
  return (
    <div>
      <h2>- {name}</h2>
      <p>{message}</p>
    </div>
  );
}
