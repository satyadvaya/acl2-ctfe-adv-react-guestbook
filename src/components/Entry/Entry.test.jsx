import React from 'react';
import { render } from '@testing-library/react';
import Entry from './Entry';

it('should render the Entry component', () => {
  const { container } = render(
    <Entry entry={{ name: 'name', message: 'message' }} />
  );

  expect(container).toMatchSnapshot();
});
