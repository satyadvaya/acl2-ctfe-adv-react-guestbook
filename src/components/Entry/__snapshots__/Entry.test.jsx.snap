// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render the Entry component 1`] = `
<div>
  <div>
    <h2>
      - 
      name
    </h2>
    <p>
      message
    </p>
  </div>
</div>
`;
